pkgs/tools/misc/bandwidth/default.nix

```diff
@@ -31,7 +51,7 @@ stdenv.mkDerivation rec {
     homepage = "https://zsmith.co/bandwidth.html";
     description = "Artificial benchmark for identifying weaknesses in the memory subsystem";
     license = licenses.gpl2Plus;
-    platforms = platforms.x86;
+    platforms = platforms.x86 ++ platforms.arm ++ platforms.aarch64;
```

Comment:
This looks more like unix or all.

Reply:
No, each platform has its own assembly routines, so e.g. risc-v and power aren't supported

Reply:
risk-v and power have such a low support level that it probably can't even compile all the dependencies of bandwhich for those platforms. Generally we don't need to consider them at this point.